dom.bs

```diff
@@ -9818,6 +9818,88 @@ the given value.
 
 
 
+<h2 id=xpath>XPath</h2>
+
+<cite>DOM Level 3 XPath</cite> defined an API for evaluating <cite>XPath 1.0</cite> expressions.
+[[DOM-Level-3-XPath]] [[XPath]] These APIs are widely implemented, but
+<cite>DOM Level 3 XPath</cite> is no longer maintained. The DOM Standard does not attempt to
+define any of the behavior, but the interface definitions are maintained here so that they can
+be updated when Web IDL changes. [[WEBIDL]]
```

Comment:
There's more changes than just that, some of them simplify the API by making arguments optional.

Let's also add something, possibly just normative, saying:

 * `XPathException` is gone (just use `DOMException`)
 * The query is matched against the DOM, and therefore contrary to the XPath 1.0 data model the root element has a parent (the Document) and text nodes can be adjacent to one another. [wilful violation]